requirements.txt

```diff
@@ -1,3 +1,4 @@
 ###### Requirements without Version Specifiers ######
-transitfeed
-pandas
```

Comment:
is this removed b/c partridge will install it for us?  do we care about version?

Reply:
Requirements are currently sprinkled across three different files. This does not seem like a good fit. I would argue we only need two.

- requirements.txt (recommend getting rid of this)
- .travis.yml (used to setup testing environment on Travis)
- setup.py (install_requires: merge and keep this one)

The requirements.txt seems duplicative. At a minimum, this should be an exhaustive list and the union of .travis.yml and setup.py.

Reply:
We need the Pandas version as <=0.20.3. Something breaks in the latest version of Pandas when I upgraded. I've spent the time to explore the issue yet.

Reply:
Check.